content/python/concepts/sets/terms/difference/difference.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.difference()'
+Description: 'Returns a new set of objects unique between two or more sets.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.difference()` method is used for identifying objects unique between two or more [sets](https://www.codecademy.com/resources/docs/python/sets) and returns a new set of those objects.
+
+## Syntax
+
+```pseudo
+set.difference(set1, set2, .., setN)
+
+setA - setB - setC
+```
+
+The following syntaxes can be used:
+
+- The `.difference()` method can be called directly against `set` with multiple sets passed as arguments.
+- Optionally, the `-` [operator](https://www.codecademy.com/resources/docs/python/operators) can be used between sets `setA`, `setB`, and `setC` where it will return the same result as the `.difference()` method.
+
+## Example
+
+The following example uses the `-` operator to create a new set of likes that are not shared between `sue_likes` and `erika_likes`:
```

Comment:
Again, this implies a different result than what we actually get. This description suggests we would get {'Tofu','Popcorn'} as a result.